### ISSUE TITLE
Expose workflow import config via API

### ISSUE THREAD

**Issue by innovate-invent** [galaxyproject/galaxy#10683]
Currently it is only possible to set a workflow to importable via the API during creation.
```
PUT /api/workflows/{id}
{ "importable": true }
```
should be added to that endpoint

Additionally, `GET /api/workflows/{id}` should return
```json
{ "importable": true, "import_path": "/u/nolan/w/myworkflow" }
```

**Comment by simonbray** [comment]
Hi @innovate-invent, I had a look at this, but it's not really clear to me. How is the setting the importable attribute different to publishing? Is it possible to set in the UI already? Can you clarify this a bit?

**Comment by innovate-invent** [comment]
If you have a look at the workflow share ui there are two options: publish and share via url. I am assuming importable refers to the latter given the logic in the workflow create function that published implies importable.

**Comment by simonbray** [comment]
Okay, makes sense. So importable = available only if you have the URL and published = listed under `workflows/list_published`.

**Comment by davelopez** [comment]
Fixed by #13571
Thank you so much @simonbray!
